src/plugins/data/public/search/search_source/search_source.ts

```diff
@@ -318,6 +315,43 @@ export class SearchSource {
    * PRIVATE APIS
    ******/
 
+  /**
+   * Run a search using the search service
```

Comment:
Sometimes you start comment with capital letter, sometimes with lower-case. I'm thinking if we should treat all these comments as grammatically correct English sentences and start them with upper-case letter and finish with full stop.

Reply:
yeah, merged your suggestions, thanks!